runtime/doc/treesitter.txt

```diff
@@ -938,6 +902,47 @@ TSHighlighter:destroy({self})                        *TSHighlighter:destroy()*
 ==============================================================================
 Lua module: vim.treesitter.languagetree          *lua-treesitter-languagetree*
 
+
+A
+
+LanguageTree
+
+contains a tree of parsers: the root treesitter parser for {lang} and any
```

Comment:
dunno what's going on with the generator or doxygen here, seems like it treats "LanguageTree" specially